lib/util/SortableSet.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+module.exports = class SortableSet extends Set {
+
+	constructor(initialIterable, defaultSort) {
+		super(initialIterable);
+		this._sortFn = defaultSort;
+		this._lastActiveSortFn = null;
+		this._isSorted = false;
```

Comment:
`_isSorted` is redundant to `_lastActiveSortFn`. `_isSorted === !!_lastActiveSortFn`.

=> `_isSorted` can be removed.